### DISCUSSION TITLE
Licensing terms?

### DISCUSSION THREAD

**Discussion by U007D** [oxidecomputer/hubris#273] [Q&A] [answered]
Hello!

What license governs `Hubris` (and `Humility`)?  I don't see the licensing terms in the repo.

Thank you for doing this!  Our industry needs to develop much more lean, well-designed, safety-conscious software IMHO.

**Comment by jacques** [comment, marked as answer]
@U007D Both are licensed under the Mozilla Public License 2.0: [hubris LICENSE](https://github.com/oxidecomputer/hubris/blob/master/LICENSE.txt) and [humility LICENSE](https://github.com/oxidecomputer/humility/blob/master/LICENSE)
